### PR TITLE
Feat/auto sync readme api reference

### PR DESCRIPTION
**Pull request by JuroUhlar**
* Using [Readme's Github action](https://docs.readme.com/main/docs/github-actions-openapi-example), we update the [Server API Reference](https://dev.fingerprint.com/reference/getevent) of every push to `main`.
* Readme API Key added to repository secrets
* Tested on our production Readme project, see Actions to examine the runs. 
* Let me know if you would prefer it as a separate action instead of tacking it onto "Publish".

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -18,8 +18,12 @@ jobs:
...
           privateKey: '${{ secrets.PRIVATE_KEY }}'
           requestId: '${{ secrets.REQUEST_ID }}'
-      - name: Deploy 🚀
+      - name: Deploy to GitHub pages 🚀📘
         uses: JamesIves/github-pages-deploy-action@8817a56e5bfec6e2b08345c81f4d422db53a2cdc
         with:
           branch: gh-pages
           folder: dist
+      - name: Deploy to Readme API Reference 🚀🦉
+        uses: readmeio/rdme@v8
```
Comment (ilfa) [inline .github/workflows/publish.yml:27]: Use commit SHA instead of tag for security reasons please

File: package.json [hunk 2]
```diff
@@ -34,6 +34,7 @@
     "js-yaml": "^4.1.0",
     "json-loader": "^0.5.7",
     "prettier": "^2.8.0",
+    "rdme": "^8.6.4",
```
Comment (ilfa) [inline package.json:37]: Do we really need this dependency when we are using their action?

**Review by ilfa — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (JuroUhlar) [reply package.json]: Good point, we don't, removed :)

File: .github/workflows/publish.yml (hunk 1 above)
Comment (JuroUhlar) [reply .github/workflows/publish.yml]: Done, thanks :)

**Review by ilfa — APPROVED** [review]

**Pull request merged**